src/cocotb/_init.py

```diff
@@ -289,3 +291,22 @@ def _setup_regression_manager() -> None:
     elif test_filter_str:
         cocotb.regression_manager.add_filters(test_filter_str)
         cocotb.regression_manager.set_mode(RegressionMode.TESTCASE)
+
+
+def _setup_resolve_x() -> None:
+    envvar = os.getenv("COCOTB_RESOLVE_X", None)
+    try:
+        resolver = cast(
+            Union[ResolverLiteral, None],
+            {
+                None: None,
+                "VALUE_ERROR": None,
+                "ZEROS": "zeros",
+                "ONES": "ones",
+                "RANDOM": "random",
+            }[envvar],
+        )
+    except KeyError:
+        raise ValueError(f"Invalid COCOTB_RESOLVE_X value {envvar!r}") from None
```

Comment:
```suggestion
        raise ValueError(f"Invalid COCOTB_RESOLVE_X value {envvar!r}. Valid options: ...") from None
```

Let's help users to find the right keyword without having to dig up the docs.